### PR TITLE
Fix openAPIV3Schema validation issue

### LINKED ISSUE #993: Legacy chart CRD could not be applied to old Kubernetes clusters, like 1.15.12
```
$ pwd /Users/zhouzhengxi/Programming/golang/src/github.com/oam-dev/kubevela
$ kubectl apply -f legacy/charts/vela-core-legacy/crds/
Error: failed to install CRD crds/standard.oam.dev_routes.yaml: CustomResourceDefinition.apiextensions.k8s.io "routes.standard.oam.dev" is invalid: 

spec.validation.openAPIV3Schema.properties[spec].properties[tls].properties[type].default: Forbidden: must not be set
```

### PR DESCRIPTION
**Pull request by zzxwill**
Legacy crd charts/vela-core/crds/standard.oam.dev_routes.yaml could not
be applied to old Kubernetes clusters like 1.15.12.
Fix #993

### CODE REVIEW

File: charts/vela-core/crds/standard.oam.dev_routes.yaml [hunk 1]
```diff
@@ -116,7 +116,6 @@ spec:
                   issuerName:
                     type: string
                   type:
-                    default: Issuer
```
Comment (wonderflow) [inline charts/vela-core/crds/standard.oam.dev_routes.yaml:119]: we should delete the legacy crd dir, this crd will not install in target k8s 1.15

Comment (zzxwill) [reply charts/vela-core/crds/standard.oam.dev_routes.yaml:119]: How about the others?

legacy/charts/vela-core-legacy/crds/standard.oam.dev_routes.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_rollouttraits.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_rollouts.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_autoscalers.yaml
legacy/charts/vela-core-legacy/crds/standard.oam.dev_metricstraits.yaml

**Review by wonderflow — APPROVED** [review]
LGTM, one question, why delete `charts/vela-core/crds/standard.oam.dev_rollouts.yaml`?

**Review by ryanzhang-oss — APPROVED** [review]

**Comment by zzxwill** [comment]
> LGTM, one question, why delete `charts/vela-core/crds/standard.oam.dev_rollouts.yaml`?

It could not be generated by `make manifests` and it's not like a base CRD like cert-manger or Prometheus. I confirmed the right CRD should be `rollouttraits`. So I deleted it. @ryanzhang-oss Please help double confirm it:)

**Pull request merged**